.env.production.template

```
@@ -1,171 +1,54 @@
-# 5. EMAIL SYSTEM (Google Workspace - CIT.EDU.LY)
-# -----------------------------------------------
-SMTP_HOST=smtp.gmail.com
+# ===========================================================
+# EBIC Website - Production Environment Template
+# ===========================================================
+# Copy this file to .env and fill in all values.
+# NEVER commit .env to version control.
+# ===========================================================
+
+# --- Database ---
+DATABASE_URL="mysql://user:password@host:3306/citcoder_eitdc"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)